acm/acm_batch_invoice/models/acm_batch_invoice.py

```diff
@@ -365,11 +365,21 @@ class ACMBatchInvoiceLine(models.Model):
     def _compute_all_amount(self):
         utility_types = ['water_amount', 'electric_amount',
                          'electric_amount_2', 'flat_rate']
+        is_minimum_check = self.env['ir.config_parameter'].sudo().get_param('minimum.check')
+        minimum_electric_amount = float(self.env['ir.config_parameter'].sudo().get_param('minimum.electric.amount'))
+        minimum_water_amount = float(self.env['ir.config_parameter'].sudo().get_param('minimum.water.amount'))
```

Comment:
No need for is_minimum_check, just check from minimum_electric_amount, minimum_water_amount is enough

**better code**
```
param = self.env['ir.config_parameter'].sudo().get_param
minimum_electric_amount = float(param('minimum.electric.amount', 0.0))
minimum_water_amount = float(param('minimum.water.amount', 0.0))
```